appgate/syncer/operator.py

```diff
@@ -83,14 +82,11 @@ def git_operator_context(
         timeout=int(os.getenv(TIMEOUT_ENV) or args.timeout),
         target_tags=get_tags(args.target_tags, os.getenv(APPGATE_TARGET_TAGS_ENV)),
         dry_run=dry_run_mode,
-        git_vendor=ensure_env(GIT_VENDOR_ENV),
+        git_vendor=get_git_vendor(ensure_env(GIT_VENDOR_ENV)),
         git_repository=ensure_env(GIT_REPOSITORY_ENV),
         git_base_branch=ensure_env(GIT_BASE_BRANCH_ENV),
         log_level=os.environ.get(APPGATE_LOG_LEVEL, "info"),
         git_repository_fork=os.environ.get(GIT_REPOSITORY_FORK_ENV),
-        main_branch=os.environ.get(
```

Comment:
mmmm what happens in case of forked repos like this:

```
base repo user1/repo1 branch A
working repo user2/repo1 branch B
```

or 

```
base repo user1/repo1 branch A
working repo user1/repo2 branch B
```

In that case base != main right?